### ISSUE TITLE
Text field disappear in some situations

### ISSUE THREAD

**Issue by andyleekp** [YousicianGit/UnityNativeEdit#55]
I found that there is a problem in mobile environment.

In some kind of situations such as sliding the task/navigation bar or switching other apps and go back my app again on Andriod or iOS, all the text fields in the input fields which attached with the script of "Native Edit Box" has gone. But when I reload the scene or set the game object to inactive and active again, the text fields appears again!

Is there any missing thing in the script or we can do so that to fix this problem? I think the text fields actually are still here but just go somewhere, may be out of camera or something? Thanks for attention~

**Comment by bthdimension** [comment]
The same thing happens to me on Android (on iOS there is no such problem). Did you ever find a solution for the issue?

**Comment by andyleekp** [comment]
Yes, we can use **OnApplicationFocus** and **OnApplicationPause** to fix/make the native text field appear again.

When we switch between apps or slide the status bar on the top of iOS/Android, the status of our app is "**PAUSED**" and OnApplicationPause will be called at the moment. Therefore, we can just set the native input field object to inactive (_setActive(false)_) in OnApplicationPause. When we switch back to our app, OnAppliactionFocus will be called again and we can set the native input field object to active (_setActive(true)_) again. The input text field will appear again.

**Comment by bthdimension** [comment]
Thanks, I'll try that :-)

**Comment by Nezz** [comment]
We are using a method similar to what @andyleekp described, but only with `OnApplicationFocus`. Feel free to contribute by submitting a pull request which integrates this into the plugin.
